src/lib.rs

```diff
@@ -0,0 +1,97 @@
+#![cfg_attr(not(test), no_std)]
```

Comment:
We probably want our standard MPL boilerplate here:
```rust
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this
// file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Reply:
True! done.